client/src/pages/layouts/AppLayout.js

```diff
@@ -71,11 +74,9 @@ const AppLayout = ({children}) => {
             </Box>
           </Box>
         </Box>
-        <Box flexGrow={1}>
-          <NavBar />
-        </Box>
+        <NavBar />
       </Box>
-      <Box flexGrow={1}>{children}</Box>
+      <CenteringBox flexGrow={1}>{children}</CenteringBox>
```

Comment:
I'm a bit on the fence about this change. I had the exact same thought while making the layout and I opted out of it because I wanted to let content have freedom as to choose where to put its elements (i.e. the content can wrap it's elements with a stretched `CenteringBox` if desired). That said, it is really difficult to imagine a scenario where we don't want the content centered and even if so, same solution applies (i.e. wrap with a stretched "regular" `Box`). Would like to other opinions as well.

Reply:
I think with what we have rn and current pipeline of features, this is fine. But in the long-run, I think giving more freedom to inner components is the right move for other possible features such as checking for board changes history where we would have change description on the right and on the left is an squished in adjusted "old version of the board". There we would like to have more flexibility on how to pivot the columns relative to the changes-list.

Reply:
Initially when I wanted to render my board, it would be offset at the bottom left corner. After I merged the layout and saw this Centering Box I just decided to use it to center the board to make it easier to interact with since my window screens are split often on my desktop. I concur with freely moving components but I didn't want to stay too distracted by layout when I was doing dnd. I think much of the layout and css and folder structure are something to discuss later as a team. =]